lib/faker/address.ex

```diff
@@ -9,146 +9,1232 @@ defmodule Faker.Address do
 
   @doc """
   Return random building number.
+
+  ## Examples
+
+      iex> Faker.Address.building_number()
+      #=> "99"
   """
-  @spec building_number() :: String.t
+  @spec building_number() :: String.t()
```

Comment:
I'm unsure about these additions of `()` in the `@spec`s at many places (but not everywhere) - are these needed for some reason?